common/test/java/com/couchbase/lite/DbCollectionsTest.kt

```diff
@@ -333,4 +382,29 @@ class DbCollectionsTest : BaseCollectionTest() {
 
         recreateBastTestDb()
     }
-}
\ No newline at end of file
+
+    /**
+     * Collections and Cross Database instance
+     */
+
+    @Ignore("There is no collection on other db")
```

Comment:
It seems that this should work.  If you can't remove the Ignore, can you say what is necessary to make it work?